camelConnector/src/main/resources/artifactMap.json

```diff
@@ -1,5 +1,5 @@
 {
-    "camelVersion": "3.21.0",
+    "camelVersion": "4.4.3",
```

Comment:
Lots of component changes due to version changes.  We discovery these via camel code, so this just updates to facts.